src/ip/tools.cc

```diff
@@ -9,6 +9,8 @@
 /* DEBUG: section 21    Misc Functions */
 
 #include "squid.h"
+#include "compat/socket.h"
+#include "compat/unistd.h"
```

Comment:
With these you can remove the redundant `unistd.h` and `sys/socket.h` includes below.

Reply:
both removed